ubuntu/build_and_push_images.sh

```diff
@@ -5,7 +5,9 @@ if [[ $EUID -ne 0 ]]; then
    exit 1
 fi
 
-apt-get -qq --yes --no-install-recommends install whois debootstrap apt-cacher-ng 2>/dev/null
+apt-get -qq --yes --no-install-recommends install whois debootstrap 2>/dev/null
```

Comment:
Actually, do you need any of these utilities (whois, debootstrap) on the provisioner at all?